/*package com.livraria.bookstore.exception;


import static org.junit.jupiter.api.Assertions.assertThrows;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;



@SpringBootTest
@RunWith(SpringRunner.class)
public class BadRequestExceptionTest {

    @Test
    public void testTest() throws Exception{
        assertThrows(BadRequestException.class,() -> new BadRequestException("Test"));
    }
}*/